configs/kubernetes-configmap.yaml

```diff
@@ -3,6 +3,8 @@ metadata:
   name: publisher-config
 data:
   config: |
+    repo-org: kubernetes
```

Comment:
as written above, I don't like the `repo-` prefix. Why not `source-org` and `source-repo`? Matches `target-org` below.